activesupport/lib/active_support/core_ext/date_and_time/calculations.rb

```diff
@@ -31,6 +31,18 @@ def today?
       to_date == ::Date.current
     end
 
+    # Returns true if the date/time is tomorrow.
+    def tomorrow?
+      to_date == ::Date.current.tomorrow
+    end
+    alias_method :next_day?, :tomorrow?
```

Comment:
Can you use a bit faster `alias` syntax than `alias_method` method for the literal alias?

```suggestion
    alias :next_day :tomorrow?
```

Reply:
Sure, replacing all usage of `alias_method`with `alias`